<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteFileDlg</class>
 <widget class="QWidget" name="RemoteFileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RemoteFileDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titleBar_statusLab">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>12</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleBar_titleLab">
        <property name="text">
         <string>远程文件</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>530</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="titleBar_minBtn">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>—</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="titleBar_maxBtn">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="titleBar_closeBtn">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="leftMargin">
       <number>21</number>
      </property>
      <property name="rightMargin">
       <number>21</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>本地设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>远程设备</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>23</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>21</number>
        </property>
        <property name="rightMargin">
         <number>21</number>
        </property>
        <item>
         <widget class="QPushButton" name="localFile_backBtn">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>返回上一层</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="localFile_refreshBtn">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>刷新当前文件夹</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="localFile_newFolderBtn">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="toolTip">
           <string>新建文件夹</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="localFile_curPathCmb">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="transferBtn">
       <property name="minimumSize">
        <size>
         <width>74</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>74</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>传输</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>21</number>
        </property>
        <property name="rightMargin">
         <number>21</number>
        </property>
        <item>
         <widget class="QPushButton" name="remoteFile_backBtn">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remoteFile_refreshBtn">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remoteFile_newFolderBtn">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="remoteFile_curPathCmb">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>21</number>
     </property>
     <item>
      <widget class="QTableView" name="localFileTableView"/>
     </item>
     <item>
      <widget class="QTableView" name="remoteFileTableView"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="allStartBtn">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>开始全部</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="allStopBtn">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>暂停全部</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="allCancelBtn">
        <property name="minimumSize">
         <size>
          <width>93</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>取消全部</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>419</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="transferDetailTableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
